components/x-follow-button/src/styles/mixins/lozenge/_themes.scss

```diff
@@ -6,28 +6,32 @@ $myft-lozenge-themes: (
 		text: oColorsByName('white'),
 		highlight: oColorsByName('claret-50'),
 		pressed-highlight: rgba(oColorsByName('black'), 0.05),
-		disabled: rgba(oColorsByName('black'), 0.5)
+		disabled: rgba(oColorsByName('black'), 0.5),
+		focus-outline: oColorsByUsecase('focus', 'outline', $fallback: null)
```

Comment:
This colour is the same grey colour which is applied by o-nomalise ([here](https://github.com/Financial-Times/o-normalise/blob/dce66be26c3a4d554044877a5f061a161efd2079/src/scss/_variables.scss#L5))